contracts/Governance.sol

```diff
@@ -176,13 +183,17 @@ contract Governance is Context, Ownable, ERC165, EIP712 {
         proposalThresholds[DEVELOPER_ROLE] = DEFAULT_PROPOSAL_THRESHOLD;
         proposalThresholds[LEGAL_ROLE] = DEFAULT_PROPOSAL_THRESHOLD;
         proposalThresholds[TREASURY_ROLE] = DEFAULT_PROPOSAL_THRESHOLD;
+        actions[address(this)]['setTreasury(address)'] = VotingParams(
+            TREASURY_ROLE,
+            DEFAULT_PROPOSAL_THRESHOLD
+        );
     }
 
     /**
      * @dev Address through which the governor executes action. Will be overloaded by module that execute actions
      * through another contract such as a timelock.
      */
-    function _executor() internal view virtual returns (address) {
+    function executor() public view virtual returns (address) {
```

Comment:
question: can this go back to being `internal`?

Reply:
Np